Can't send an Embed Message if including a Locally based image as Author Icon

When this is executed
```
new MessageBuilder()
                        .setEmbed(new EmbedBuilder()
                                .setAuthor("Name", "https://google.com/", Main.class.getResourceAsStream("/about/icon.png"))
                        )
                        .send(event.getChannel()).exceptionally(ExceptionLogger.get());
```
It will throw this error
Quick note: The `Main.class.getResourceAsStream("/about/icon.png")` will work for the thumbnail without any issues
```
2018-08-29 14:53:46.390-0500 ERROR org.javacord.core.util.logging.ExceptionLoggerDelegateImpl Caught unhandled exception! {shard=0}
 java.util.concurrent.CompletionException: Received a 400 response from Discord with body {"embed": ["author"]}!
        at me.orangeflare.openyuri.Main.lambda$core$2(Main.java:71)
        at org.javacord.core.util.event.EventDispatcher.lambda$dispatchMessageCreateEvent$93(EventDispatcher.java:2499)
        at org.javacord.core.util.event.EventDispatcherBase.lambda$dispatchEvent$10(EventDispatcherBase.java:191)
        at org.javacord.core.util.event.EventDispatcherBase.lambda$checkRunningListenersAndStartIfPossible$21(EventDispatcherBase.java:270)
        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
        at java.util.concurrent.FutureTask.run(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
        at java.lang.Thread.run(Unknown Source)
Caused by: org.javacord.api.exception.BadRequestException: Received a 400 response from Discord with body {"embed": ["author"]}!
        at org.javacord.core.util.rest.RestRequestHttpResponseCode.lambda$getDiscordException$2(RestRequestHttpResponseCode.java:224)
        at java.util.Optional.map(Unknown Source)
        at org.javacord.core.util.rest.RestRequestHttpResponseCode.getDiscordException(RestRequestHttpResponseCode.java:224)
        at org.javacord.core.util.rest.RestRequest.lambda$executeBlocking$10(RestRequest.java:415)
        at java.util.Optional.flatMap(Unknown Source)
        at org.javacord.core.util.rest.RestRequest.executeBlocking(RestRequest.java:414)
        at org.javacord.core.util.ratelimit.RatelimitManager.lambda$queueRequest$8(RatelimitManager.java:150)
        ... 5 more
Caused by: java.lang.Exception: origin of RestRequest call
        at org.javacord.core.util.rest.RestRequest.<init>(RestRequest.java:79)
        at org.javacord.core.entity.message.MessageBuilderDelegateImpl.send(MessageBuilderDelegateImpl.java:241)
        at org.javacord.api.entity.message.MessageBuilder.send(MessageBuilder.java:318)
        at me.orangeflare.openyuri.Main.lambda$core$2(Main.java:71)
        at org.javacord.core.util.event.EventDispatcher.lambda$dispatchMessageCreateEvent$93(EventDispatcher.java:2499)
        at org.javacord.core.util.event.EventDispatcherBase.lambda$dispatchEvent$10(EventDispatcherBase.java:191)
        at org.javacord.core.util.event.EventDispatcherBase.lambda$checkRunningListenersAndStartIfPossible$21(EventDispatcherBase.java:270)
        ... 5 more
```

--- a/javacord-core/src/main/java/org/javacord/core/entity/message/embed/EmbedBuilderDelegateImpl.java
+++ b/javacord-core/src/main/java/org/javacord/core/entity/message/embed/EmbedBuilderDelegateImpl.java
@@ -532,7 +532,7 @@ public class EmbedBuilderDelegateImpl implements EmbedBuilderDelegate {
                 author.put("icon_url", authorIconUrl);
             }
             if (authorIconContainer != null) {
-                author.put("url", "attachment://" + authorIconContainer.getFileTypeOrName());
+                author.put("icon_url", "attachment://" + authorIconContainer.getFileTypeOrName());
             }
         }
         if (thumbnailUrl != null && !thumbnailUrl.equals("")) {

